test/onnx/test_pytorch_onnx_onnxruntime.py

```diff
@@ -979,6 +979,7 @@ def forward(self, x):
 
         x = torch.randn(2, 3, 4)
         self.run_test(RandNLike(), x)
+        self.run_test(torch.jit.script(RandNLike()), x)
 
         class RandLike(torch.nn.Module):
```

Comment:
Does RandLike have the same kind of issue?

Reply:
yes it does wasn't sure if I should change in the same PR or have separate